cloud-gateway-service/src/main/java/org/zowe/apiml/cloudgatewayservice/service/GatewayIndexService.java

```diff
@@ -167,4 +170,8 @@ private boolean hasSameApiId(ServiceInfo serviceInfo, String apiId) {
         }
         return false;
     }
+
+    private boolean hasSameServiceId(ServiceInfo serviceInfo, String serviceId) {
+        return StringUtils.equals(serviceId, serviceInfo.getServiceId());
+    }
```

Comment:
is the method really needed? It's a single line and used once

Reply:
it used in stream filtering predicate `hasSameServiceId` `hasSameApimlId` makes higher order calls more readable